packages/contracts-bedrock/scripts/deploy/DeployImplementations.s.sol

```diff
@@ -17,8 +17,12 @@ import { IPreimageOracle } from "interfaces/cannon/IPreimageOracle.sol";
 import { IMIPS } from "interfaces/cannon/IMIPS.sol";
 import { IDisputeGameFactory } from "interfaces/dispute/IDisputeGameFactory.sol";
 import { IAnchorStateRegistry } from "interfaces/dispute/IAnchorStateRegistry.sol";
-import { IOPContractsManager } from "interfaces/L1/IOPContractsManager.sol";
-import { IOPContractsManagerInterop } from "interfaces/L1/IOPContractsManagerInterop.sol";
+import {
+    IOPContractsManager,
+    IOPContractsManagerGameTypeAdder,
+    IOPContractsManagerDeployer,
+    IOPContractsManagerUpgrader
```

Comment:
Do we need to split the `addGameType()` and `upgrade()` methods out into separate contracts?

I feel like having the functions that need to be delegatecalled together make sense.